docs/source/quickstart/index.rst

```diff
@@ -20,6 +20,66 @@ To install Cloud Custodian, just run::
 
   $ pip install c7n
 
+
+.. _write-policy:
+
+Write your first policy
+-----------------------
+
+A policy specifies the following items:
+
+* The resource to run the policy against
+* The resource state or states the policy checks (filters)
+* The action or actions to take on the resource depending on the specified filters
+
+For this tutorial, let's stop all EC2 instances that are tagged with
+``Custodian``. To get started, go make an EC2 instance in your `AWS console
+<https://console.aws.amazon.com/>`_, and tag it with the key ``Custodian`` (any
+value).  Also, make sure you have an access key handy.
+
+Then, create a file named ``custodian.yml`` with this content:
+
+.. code-block:: yaml
+
+    policies:
+      - name: my-first-policy
+        resource: ec2
+        filters:
+          - "tag:Custodian": present
+        actions:
+          - stop
+
+Now, run Custodian:
+
+.. code-block:: bash
+
+    AWS_ACCESS_KEY_ID="foo" AWS_SECRET_ACCESS_KEY="bar" custodian run --output=. --config=custodian.yml
+
+If you are not using the ``us-east-1`` region, then you'll need to specify that as well, like so:
+
+.. code-block:: bash
+
+    --region=us-west-1
```

Comment:
fwiw, this can also be sourced from env vars AWS_DEFAULT_REGION, and we have a bug open to source it from cli config.